Basic-Recruit-Training/Person.cpp

```diff
@@ -0,0 +1,73 @@
+#include <iostream> //Imports cout
+#include <sstream>
+#include <string>
+#include "Person.h"
+
+// Person::Person();  //creating what the functions actually DO here
+Person::Person()    {
+name_ = "";
+age_ = new int;
```

Comment:
here we are dynamically allocating memory, but not giving age any sort of default value, although it might be zero, it is a good idea to explicitly assign it a value